### PR TITLE
Channels usability

### LINKED ISSUE #4: Channel's usability
A user should have an option:
- [x] to change
- [x] to join to another channel
- [x] to create a channel

### PR DESCRIPTION
**Pull request by cristianbgp**
Add the following functions:
- createChannel
- joinChannel
- changeActiveChannel

Close #4

### CODE REVIEW

File: public/assets/js/main.js [hunk 1]
```diff
@@ -1,13 +1,44 @@
...
+    userChannels.push(channelName);
+    return userChannels;
+  }
+}
+
+function changeActiveChannel(channelName) {
+  const channelExists = userChannels.includes(channelName);
+  if (channelExists) {
+    activeChannel = channels[channelName];
+    return activeChannel;
```
Comment (yummta) [inline public/assets/js/main.js]: You could return channels[channelName] directly. Think about if in this case is necesary create activeChannel.

File: public/assets/js/main.js [hunk 2]
```diff
@@ -1,13 +1,44 @@
...
+let channels = { general: { name: "general" } };
+let userChannels = ["general"];
+let activeChannel = channels["general"];
+
+function createChannel(channelName) {
+  const channelCreated = { channelName: { name: channelName } };
+  const channelExists = channels.hasOwnProperty(channelName);
+  if (!channelExists) {
+    channels[channelName] = channelCreated.channelName;
+    joinChannel(channelName);
```
Comment (liannivin) [inline public/assets/js/main.js]: Don't forget to assign the created channel as active channel to user.

Comment (cristianbgp) [reply public/assets/js/main.js]: Do you think it is necessary?

File: public/assets/js/main.js (hunk 1 above)
Comment (cristianbgp) [reply public/assets/js/main.js]: I'll made it in one line `return activeChannel = channels[channelName];`

File: public/assets/js/main.js (hunk 2 above)
Comment (cristianbgp) [reply public/assets/js/main.js]: I already added the call for that function

**Pull request merged**